data/_ui/tooltips.txt

```diff
@@ -853,109 +853,109 @@ tip "firing burn / second:"
 
 tip "heat damage / shot:"
-	`Each shot increases the target ship's heat by this amount, or half this amount if its shields are up.`
+	`Increases the target ship's heat by this amount per shot. Only half as effective when the target's shields are up.`
 
 tip "energy damage / shot:"
-	`Each shot reduces the target ship's energy by this amount, or half this amount if its shields are up.`
+	`Reduces the target ship's energy by this amount per shot. Only half as effective when the target's shields are up.`
 
 tip "ion damage / shot:"
-	`Each shot increases the target's ionization by this amount. Ionization reduces the target's energy. Ionization slowly wears off over time.`
+	`Increases the target's ionization by this amount per shot. Only half as effective when the target's shields are up. Ionization reduces the target's energy, and slowly wears off over time.`
 
 tip "scrambling damage / shot:"
-	`Each shot increases the target's scrambling by this amount. scrambling causes weapon failures. It slowly wears off over time.`
+	`Increases the target's chance of weapon failure. Only half as effective when the target's shields are up. Scrambling slowly wears off over time.`
 
 tip "slowing damage / shot:"
-	`Reduces the target's turn rate, acceleration, and top speed. Slowing wears off over time.`
+	`Reduces the target's turn rate, acceleration, and top speed. Only half as effective when the target's shields are up. Slowing wears off over time.`
 
 tip "disruption damage / shot:"
-	`Disrupts the target's shields, allowing weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, and adding additional shield disruption becomes more difficult as damage leaks past shields instead of disrupting them.`
+	`Allows weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, and adding additional shield disruption becomes more difficult as damage leaks past shields instead of disrupting them.`
 
 tip "discharge damage / shot:"
-	`Each shot increases the target's shield discharge by this amount. Shield discharge continuously reduces the target's shields and slowly wears off over time.`
+	`Shield damage dealt over time by this weapon per shot.`
 
 tip "corrosion damage / shot:"
-	`Each shot increases the target's hull corrosion by this amount. Hull discharge continuously reduces the target's hull and slowly wears off over time.`
+	`Hull damage dealt over time by this weapon per shot.`
 
 tip "leak damage / shot:"
-	`Each shot causes the target to leak by this amount. Leak continuously decreases the target's fuel and slowly wears off over time.`
+	`Fuel damage dealt over time by this weapon per shot.`
 
 tip "burn damage / shot:"
-	`Each shot causes the target to burn by this amount. Burn continuously increases the target's heat and slowly wears off over time.`
+	`Fuel damage dealt over time by this weapon per shot.`
```

Comment:
Is this supposed to be the same as the tooltip above?

Reply:
```suggestion
	`Heat damage dealt over time by this weapon per shot.`
```